frameworks/PHP/fat-free/index.php

```diff
@@ -1,36 +1,36 @@
 <?php
 // drop startup errors
-if (function_exists('error_clear_last'))
-    error_clear_last();
+// if (function_exists('error_clear_last'))
+//     error_clear_last();
    
-error_reporting(0);
+// error_reporting(0);
 
 /** @var Base $f3 */
 $f3=require('src/base.php');
 
 error_reporting(-1);
 
-$f3->set('DEBUG',2);
+$f3->set('DEBUG',0);
 $f3->set('HIGHLIGHT',false);
 $f3->set('CACHE','folder=tmp/cache/');
 $f3->set('UI','ui/');
 $f3->set('ONERROR',function($f3){
     echo $f3->get('ERROR.code').': '.$f3->get('ERROR.text')."\n".$f3->get('ERROR.trace');
 });
 
-$f3->set('DBS',array('mysql:host=tfb-database;port=3306;dbname=hello_world','benchmarkdbuser','benchmarkdbpass',[PDO::ATTR_PERSISTENT => true]));
+$f3->set('DBS',array('mysql:host=tfb-database;port=3306;dbname=hello_world','benchmarkdbuser','benchmarkdbpass'));
```

Comment:
Without persistent db connections will have less than 1K req/s.

Reply:
we can leave that, but it's actually not being used in the rest of the file. I was just cleaning it up. It's being called directly in the other db calls.

Reply:
DRY and delete it in the other parts.
Also use variadic to simplify the code.